crypten/mpc/primitives/arithmetic.py

```diff
@@ -250,9 +250,9 @@ def _arithmetic_function(self, y, op, inplace=False, *args, **kwargs):
             else:  # ['mul', 'matmul', 'convNd', 'conv_transposeNd']
                 # NOTE: 'mul_' calls 'mul' here
                 # Must copy share.data here to support 'mul_' being inplace
-                result.share.data = getattr(beaver, op)(
-                    result, y, *args, **kwargs
-                ).share.data
+                result.share.set_(getattr(beaver, op)(
```

Comment:
I think the comment above is stale now.

Is the .data below still needed? I think in pytorch we intend to eventually deprecate `.data`.